compile.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Builds all targets and runs all tests
+if [[ ! -z "$CIRCLECI" ]]; then
+  DELETED_PACKAGES="";
+else
+  DELETED_PACKAGES="--deleted_packages $(cat .circleci/deleted_bazel_packages.txt)";
```

Comment:
What is `deleted_bazel_packages`?

Reply:
@oferb oops, I think I blindly copy-pasted this from `startup-os` where it *is* needed for ignoring `cc_binary` target that would enforce full `protoc` compilation -- which we are trying to avoid